packages/aws-cdk-lib/aws-ecs/lib/base/task-definition.ts

```diff
@@ -539,7 +535,19 @@ export class TaskDefinition extends TaskDefinitionBase {
         operatingSystemFamily: this.runtimePlatform?.operatingSystemFamily?._operatingSystemFamily,
       } : undefined,
       enableFaultInjection: props.enableFaultInjection,
-    });
+    };
+
+    if (props.inferenceAccelerators) {
+      taskDefProps = {
+        ...taskDefProps,
+        inferenceAccelerators: Lazy.any({
+          produce: () =>
+            this.renderInferenceAccelerators(),
+        }),
+      };
```

Comment:
You can simplify this to:
```suggestion
      taskDefProps.inferenceAccelerators = Lazy.any({
        produce: () =>
          this.renderInferenceAccelerators(),
      }),
```

Reply:
No, that's not possible, `taskDefProps.inferenceAccelerators` is a read-only property.